### ISSUE TITLE
Default console does not change to Command Prompt

### ISSUE THREAD

**Issue by alefiya-naseem** [microsoft/vscode#132768]
Issue Type: <b>Bug</b>

I have been trying to change my default console to "cmd.exe" from Powershell but it does not change. I have set default console as "Command Prompt" in settings and also settings.json and every other place possible. It still opens up Powershell on startup and when I run and debug. I am not able to run my conda env in Powershell.

The User settings.json file:
```
{
    "workbench.colorTheme": "Visual Studio Dark",
    "editor.suggestSelection": "first",
    "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
    "python.condaPath": "C:\\Users\\a680848\\Anaconda3\\Scripts\\conda.exe",
    "python.jediEnabled": false,
    "autoDocstring.docstringFormat": "google",
    "autoDocstring.includeExtendedSummary": true,
    "python.venvPath": "C:\\Users\\a680848\\virtualenvs",
    "C_Cpp.workspaceSymbols": "All",
    "editor.detectIndentation": false,
    "editor.rulers": [
    79, 120
    ],
    "C_Cpp.updateChannel": "Insiders",
    "python.formatting.provider": "black",
    "python.formatting.blackArgs": [
        "--line-length=79"
    ],
    "python.analysis.extraPaths": [
        "libs",
        "code/Sites/DataManagement"
    ],
    "python.analysis.typeCheckingMode": "strict",
    "jupyter.sendSelectionToInteractiveWindow": false,
    "jupyter.debugJustMyCode": false,
    "jupyter.alwaysTrustNotebooks": true,
    "cSpell.enableFiletypes": [
        "django-html"
    ],
    "python.languageServer": "Pylance",
    "python.linting.flake8CategorySeverity.F": "Warning",
    "workbench.editorAssociations": {
        "*.ipynb": "jupyter-notebook"
    },
    "cSpell.userWords": [
        "boto"
    ],
    "python.linting.flake8Enabled": true,
    "terminal.integrated.defaultProfile.windows": "Command Prompt",
    "terminal.integrated.profiles.windows": {
    
        "C:\\WINDOWS\\System32\\cmd.exe (migrated)": {
            "path": "C:\\WINDOWS\\System32\\cmd.exe",
            "args": [
                "/K",
                "C:\\Users\\a680848\\Anaconda3\\Scripts\\activate.bat C:\\Users\\a680848\\Anaconda3"
            ]
        }
    },
    "notebook.cellToolbarLocation": {
        "default": "right",
        "jupyter-notebook": "left"
    },
    "security.workspace.trust.untrustedFiles": "open",
    "window.zoomLevel": 1,
    "python.autoComplete.extraPaths": ["libs", "code/Sites/DataManagement"]
}
```
My VScode settings.json contains:
```
{
​​​​​   "python.pythonPath":"C:\\Users\\a680848\\Anaconda3\\envs\\website_test\\python.exe",
    "editor.rulers":[
    79,
    120
    ],
    "git.ignoreLimitWarning":true,
    "python.testing.unittestArgs":[
    "-v",
    "-s",
    ".",
    "-p",
    "test_*.py"
    ],
    "python.testing.pytestEnabled":false,
    "python.testing.nosetestsEnabled":false,
    "python.testing.unittestEnabled":false,
    "python.testing.promptToConfigure":false,
    "python.analysis.extraPaths":["libs","code/Sites/DataManagement"],
    "python.autoComplete.extraPaths": ["libs", "code/Sites/DataManagement"]
}​​​​​​​​
```

VS Code version: Code 1.60.0 (e7d7e9a9348e6a8cc8c03f877d39cb72e5dfb1ff, 2021-09-01T10:41:52.311Z)
OS version: Windows_NT x64 10.0.18363
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz (4 x 2496)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.88GB (20.22GB free)|
|Process Argv|--crash-reporter-id 4de57417-58cf-4315-bcef-a38ac441d6cb|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-django|bat|1.6.0
django|big|1.0.2
python-extension-pack|don|1.6.0
gitlens|eam|11.6.0
gc-excelviewer|Gra|3.0.44
python|ms-|2021.9.1191016588
vscode-pylance|ms-|2021.9.1
jupyter|ms-|2021.8.2041215044
jupyter-keymap|ms-|1.0.0
remote-wsl|ms-|0.58.2
vsliveshare|ms-|1.0.4761
autodocstring|njp|0.5.4
vscodeintellicode|Vis|1.2.14
jinja|who|0.0.8

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30358480
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805:30301674
pythonvspyt200:30340761
vscextlangct:30333562
binariesv615:30325510
vsccppwt:30329788
pythonvssor306:30344512
bridge0708:30335490
pygetstartedc2:30360494
bridge0723:30353136
pythonf5test824cf:30361778
javagetstartedc:30351792
pythonvspyt187:30362782
pydsgst2:30361792
vscus224cf:30358040
vssid140cf:30363604

```

</details>

**Comment by Nelik** [comment]
Started to having the same issue since today (Sep 09). Except I wanted my default terminal to be Git Bash but Visual Code was opening Power Shell. When I wanted to run tests with jest it also opened Power Shell instead of Git Bash. I changed default terminal in .json setting and in UI setting too. Nothing helped. 
I had to downgrade Visual Studio Code to version 1.59.1.
![image](https://user-images.githubusercontent.com/8331413/132728828-74022a71-28c3-484e-b625-6c0c8d9ac099.png)
